### ISSUE TITLE
[Feature Request] Continuous horizontal scrolling

### ISSUE THREAD

**Issue by grenzionky** [inorichi/tachiyomi#2291]
It would be nice if Tachiyomi had continuous scroll feature (like webtoon view), but sideways/horizontal.  
I want it specifically for right to left scrolling, but having it both ways would be nice for people reading in other languages.

**Comment by joseph619** [comment]
I would like this as well

**Comment by Dangelman** [comment]
webtoon horizontal good

**Comment by grenzionky** [comment]
> webtoon horizontal good

what does this mean? can you rephrase it please

**Comment by CarlosEsco** [comment]
he's agreeing with your ticket

**Comment by Vaturobi** [comment]
Please elaborate continuous scroll feature (like webtoon view), but sideways/horizontal.
from what i can understand you want it right to left scrolling which can be changed in settings in the reader section

**Comment by grenzionky** [comment]
what i want is to have an option that lets the scrolling be: instead of stopping between each page, you can continuously scroll even between pages. (just like how webtoon view does it, but horizontally instead of vertically). i want right to left, but it would be nice to have left to right too for other ppl

**Comment by rraboy** [comment]
it is already in the app. Settings->Default viewer->Webtoon

**Comment by grenzionky** [comment]
webtoon view scrolls vertically, i want to be able to scroll horizontally

**Comment by Habibster** [comment]
literally just came onto here to request the same thing, please someone make this a feature! when using the mangaplus, double spreads arent shown correctly (so you have to go to the next page to see the other half of the spread). continuous horizontal scrolling would (sorta) fix this!

**Comment by hXtreme** [comment]
This would also be really good addition for when you have cover spreads that span multiple pages.

**Comment by Paraless47** [comment]
I'd love this too, would be nice for when there's double-page images in manga.
